src/Umbraco.Web/Models/Mapping/TabsAndPropertiesMapper.cs

```diff
@@ -106,7 +106,7 @@ protected virtual void MapGenericProperties(IContentBase content, List<Tab<Conte
         protected virtual List<ContentPropertyDisplay> MapProperties(IContentBase content, List<Property> properties, MapperContext context)
         {
             // must pass the context through
-            return properties.OrderBy(x => x.PropertyType.SortOrder).Select(x => context.Mapper.Map<ContentPropertyDisplay>(x, context)).ToList();
+            return properties.OrderBy(x => x.PropertyType.SortOrder).Select(x => context.Map<ContentPropertyDisplay>(x)).ToList();
```

Comment:
Could be written as `return properties.OrderBy(x => x.PropertyType.SortOrder).Select(context.Map<ContentPropertyDisplay>).ToList();` I think? And then the comment above could be removed.

Reply:
totally, pushed a fix